client/src/components/lobby/PublicLobbyList.tsx

```diff
@@ -90,7 +90,7 @@ export function PublicLobbyList(props: PublicLobbyListProps) {
                         <div className={"flex items-center gap-1 text-xxs"}>
                           <ClockIcon className="h-4 w-4 text-secondary-700" />
                           <span className={"max-w-[160px] text-ellipsis overflow-hidden whitespace-nowrap"}>{`${dayjs(
-                            lobby.createdAt
+                            lobby.createdAt,
```

Comment:
Is this intentional? Seems like it might cause an error?

Reply:
the comma?